### PR TITLE
tidal: set feedback parameter to zero by default

### PR DESCRIPTION
**Pull request by telephon**
Not sure if it is still necessary, but here it is.

### CODE REVIEW

**Comment by telephon** [comment]
@lennart All in all, wouldn't it be better to change them all to `Nothing` and then keep the defaults in classic dirt and superdirt respectively?

**Comment by lennart** [comment]
as of now, specifying a param with default `Nothing` will make these mandatory. which is right now just `s`.

I don't know why there is this restriction.

**Comment by bgold-cosmos** [comment]
Dirt doesn't receive messages the same way SuperDirt does - it expects OSC messages where all parameters are in a certain order and have values.  In fact, I think this is the only reason for the "defaults" in Param.hs these days, my understanding is that SuperDirt will never sees those defaults.

My recollection is that `-1` defaults were a way of working around whether a parameter had been used in Tidal or not, equivalent to how SuperDirt can test for `nil`.

**Comment by telephon** [comment]
ok, so we can actually jut leave it as it is? Should I close?

File: Sound/Tidal/Params.hs [hunk 1]
```diff
@@ -114,7 +114,7 @@ Using `cut "0"` is effectively _no_ cutgroup.
 -- | a pattern of numbers from 0 to 1. Sets the level of the delay signal.
 (delay, delay_p)                 = pF "delay" (Just 0)
 -- | a pattern of numbers from 0 to 1. Sets the amount of delay feedback.
-(delayfeedback, delayfeedback_p) = pF "delayfeedback" (Just (-1))
+(delayfeedback, delayfeedback_p) = pF "delayfeedback" (Just 0)
```
Comment (lennart) [inline Sound/Tidal/Params.hs]: just read code again, we cannot change defaults without changing classic dirt.

**Comment by lennart** [comment]
currently we cannot change this without changing classic dirt as well, do we **want** to change classic dirt as well?

**Comment by telephon** [comment]
@lennart we want no feedback unless specified. This currently works on both dirts, I suppose, doesn't it?

**Comment by yaxu** [comment]
@telephon the problem is that delay is a global effect. If pattern a doesn't set `delayfeedback`, it would turn off feedback for a sound made with pattern b.
Could patch classic dirt to ignore `delayfeedback` if `delay` is set to 0, but why change this if superdirt ignores the default anyway?

**Pull request closed without merging**